classes/order/Order.php

```diff
@@ -699,7 +699,7 @@ protected function setProductCustomizedDatas(&$product, $customized_datas)
      *
      * @param array &$product
      */
-    protected function setProductCurrentStock(&$product)
```

Comment:
this will break the compatibility: you can create a new protected method and deprecate this one if you think the naming of the former one is bad.

Reply:
First I considered adding a new protected method and keeping this one but the behavior change is too important, I think it is better to embark it inside the existing function so that developers using it do not have "lose" the feature :) 

Incoming in next commit